### PR TITLE
created cs while loop slidedeck

### PR DESCRIPTION
**Pull request by wandercode**
image
image

### CODE REVIEW

**Comment by wandercode** [comment]
image

**Review by sneha-brainchild — COMMENTED** [review]
We can some example and output also

File: 00n_Whileloop.html [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+</head>
+
+<body>
+    <div class="reveal">
+        <div class="slides">
+            <section>
+                <h3>C# While Loop</h3>
+                <p>The while statement executes a statement or a block of statements while a specified Boolean
+                    expression evaluates to true</p>
+                <img src="/image/cswhileloopone.png" alt="cs while loop image">
```
Comment (VishnuKulakkattu) [inline 00n_Whileloop.html:25]: Please change this image to code

**Review by VishnuKulakkattu — DISMISSED** [review]

File: 00n_Whileloop.html (hunk 1 above)
Comment (wandercode) [reply 00n_Whileloop.html]: done

**Review by sneha-brainchild — APPROVED** [review]

**Review by sneha-brainchild — APPROVED** [review]

**Review by VishnuKulakkattu — APPROVED** [review]

**Pull request merged**